core/base-service/redirector.js

```diff
@@ -76,12 +78,21 @@ module.exports = function redirector(attrs) {
         trace.logTrace('inbound', emojic.ticket, 'Named params', namedParams)
         trace.logTrace('inbound', emojic.crayon, 'Query params', queryParams)
 
-        const targetUrl = target(namedParams)
+        const targetUrl = transformPath(namedParams)
```

Comment:
```suggestion
        const targetPath = transformPath(namedParams)
```

Reply:
👍 I've still yet to figure out how to  suggest changes via this new GH feature (def. a PEBCAK issue though 😆)